common/src/main/java/org/cloudfoundry/identity/uaa/zone/IdentityZoneResolvingFilter.java

```diff
@@ -86,11 +91,13 @@ public void setIdentityZoneProvisioning(IdentityZoneProvisioning dao) {
     public void setAdditionalInternalHostnames(Set<String> hostnames) {
         if (hostnames!=null) {
             this.defaultZoneHostnames.addAll(hostnames);
+            logger.info(String.format("additional internal hostnames: %s", hostnames));
```

Comment:
debug for sure here, this is not informative. I would also recommend doing 

    Arrays.toString(hostnames)